src/lib/sectionList/listViews/sectionListViewsConfig.ts

```diff
@@ -875,4 +875,54 @@ export const modelListViewsConfig = {
             overrideDefaultAvailable: true,
         },
     },
+    predictor: {
+        columns: {
+            default: [
+                DESCRIPTORS.name,
+                { label: i18n.t('Output'), path: 'output.displayName' },
+                { label: i18n.t('Period type'), path: 'periodType' },
+                'lastUpdated',
+            ],
+            available: [
+                DESCRIPTORS.shortName,
+                'code',
+                'id',
+                'href',
+                'created',
+                'createdBy',
+                'lastUpdatedBy',
+                'description',
+                DESCRIPTORS.publicAccess,
+                DESCRIPTORS.owner,
+                {
+                    label: i18n.t('Output option combo'),
+                    path: 'outputCombo.displayName',
+                },
+                { label: i18n.t('Predictor groups'), path: 'predictorGroups' },
+                {
+                    label: i18n.t('Sequential sample count'),
+                    path: 'sequentialSampleCount',
+                },
+                {
+                    label: i18n.t('Annual sample count'),
+                    path: 'annualSampleCount',
+                },
+                {
+                    label: i18n.t('Sequential skip count'),
+                    path: 'sequentialSkipCount',
+                },
+                {
+                    label: i18n.t('Organisation unit providing data'),
+                    path: 'organisationUnitDescendants',
```

Comment:
you might want to add translations for these enum values (into getConstantTranslations). Right now, we just get the enum back `DESCENDANTS`, `SELECTED`.